ovos_plugin_manager/templates/stt.py

```diff
@@ -119,21 +118,24 @@ def __init__(self):
         super().__init__()
         self.stream = None
         self.can_stream = True
+        self.transcript_ready = Event()
 
     def stream_start(self, language=None):
         self.stream_stop()
-        language = language or self.lang
         self.queue = Queue()
         self.stream = self.create_streaming_thread()
+        self.stream.language = language or self.lang
+        self.transcript_ready.clear()
         self.stream.start()
 
     def stream_data(self, data):
         self.queue.put(data)
 
     def stream_stop(self):
         if self.stream is not None:
-            text = self.stream.finalize()
+            self.transcript_ready.set()
```

Comment:
should this be after `finalize()`? I thought that method could be overridden to modify the transcriptions before return